modules/imgproc/src/smooth.cpp

```diff
@@ -1360,7 +1582,8 @@ cv::Ptr<cv::FilterEngine> cv::createBoxFilter( int srcType, int dstType, Size ks
            srcType, dstType, sumType, borderType );
 }
 
-#if defined(HAVE_IPP)
+// TODO: IPP performance regression
+#if 0 && defined(HAVE_IPP)
```

Comment:
You should [use](https://github.com/opencv/opencv/search?utf8=%E2%9C%93&q=IPP_DISABLE_BLOCK) [IPP_DISABLE_BLOCK](https://github.com/opencv/opencv/blob/ca4b7e1e0fdd064ea39e14c7b9b51cae54db5dfe/modules/core/include/opencv2/core/private.hpp#L214) instead of 0 to disable IPP optimized code for performance or accuracy reason (but this code should be able to compile).
It is useful to run all test cases with enabled IPP to track progress in case of updating IPPICV.